packages/components/src/__actions__/Button/v3/_docs/Button.spec.stories.tsx

```diff
@@ -149,16 +129,13 @@ export const ButtonWithRACRenderPropsAsChildren: Story = {
   },
 }
 
-// While this is a less likely use case, but as it is possible to use render props to apply a class name to the button so we should test that it works with our Button implementation
-export const ButtonWithRACRenderPropsAsClassname: Story = {
-  render: ({ children: _, ...otherArgs }) => (
+export const RACRenderPropsWithClassName: Story = {
+  render: ({ children, ...otherArgs }) => (
     <Button
-      className={({ isFocused }) =>
-        isFocused ? "!bg-blue-500 !text-white !border-transparent" : ""
-      }
       {...otherArgs}
+      className={({ isFocusVisible }) => (isFocusVisible ? "!bg-gray-300" : "")}
     >
-      Label
+      {children}
     </Button>
```

Comment:
```suggestion
export const RACRenderPropsWithClassName: Story = {
  render: args => (
    <Button {...args} className={({ isFocusVisible }) => (isFocusVisible ? "!bg-gray-300" : "")} />
```

Actually, this should technically work?
```suggestion
export const RACRenderPropsWithClassName: Story = {
  args: {
    className: ({ isFocusVisible }) => (isFocusVisible ? "!bg-gray-300" : "")
  }
}
```

Reply:
Nice that is cleaner 👨‍🍳 👍